webview_panels/src/uiCore/components/button/Button.tsx

```diff
@@ -1,8 +1,42 @@
+import { useState, useEffect } from "react";
 import { Button as ReactStrapButton, ButtonProps } from "reactstrap";
 import Tooltip from "../tooltip/Tooltip";
 
-export const Button = (props: ButtonProps): JSX.Element => (
-  <Tooltip title={props.title}>
-    <ReactStrapButton {...props} />
-  </Tooltip>
-);
\ No newline at end of file
+interface CustomButtonProps extends ButtonProps {
+  icon?: React.ReactNode;
+  showtext?: boolean;
```

Comment:
Change this component in this way:
- instead of buttonText props, it should be children. This will take care of all existing buttons
- if icon props is set, then it should show in new format, if not, old format
- no need for showtext props
- title is already part of buttonprops

Reply:
noted